test/e2e/storage/pvc_storageclass.go

```diff
@@ -18,172 +18,177 @@ package storage
 
 import (
 	"context"
-	"fmt"
 	"time"
 
+	"k8s.io/klog/v2"
+
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
-
 	v1 "k8s.io/api/core/v1"
 	storagev1 "k8s.io/api/storage/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 	clientset "k8s.io/client-go/kubernetes"
 	storageutil "k8s.io/kubernetes/pkg/apis/storage/util"
+	"k8s.io/kubernetes/test/e2e/feature"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2epv "k8s.io/kubernetes/test/e2e/framework/pv"
-	"k8s.io/kubernetes/test/e2e/storage/testsuites"
 	"k8s.io/kubernetes/test/e2e/storage/utils"
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = utils.SIGDescribe("Persistent Volume Claim and StorageClass", func() {
-	f := framework.NewDefaultFramework("pvc-retroactive-storageclass")
+var _ = utils.SIGDescribe("Retroactive StorageClass Assignment", feature.Kind, func() {
```

Comment:
@RomanBednar @torredil  which job will be triggered to run this e2e if this label is applied? 

I cannot find a job from https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

Reply:
This e2e test will be exercised by `pull-kubernetes-e2e-storage-kind-disruptive`  (done in this PR) and will also be periodically exercised in [e2e-ci-kubernetes-e2e-cos-gce-serial-canary](https://prow.k8s.io/job-history/gs/kubernetes-jenkins/logs/e2e-ci-kubernetes-e2e-cos-gce-serial-canary).

Reply:
`feature.Kind` is removed now in latest rev.